### DISCUSSION TITLE
Creating a draft release using electron-builder

### DISCUSSION THREAD

**Discussion by JonHollander** [keygen-sh/community#142] [General]
Hi- 
I'm using CI to build my electron app and publish releases to keygen. Is there any way to configure electron-builder to create a draft release instead of automatically publishing the new release immediately?

Thanks,
Jon

**Comment by ezekg** [comment]
If I remember correctly, electron-builder has no concept of a draft vs published release right now, so new releases [are created already published](https://github.com/electron-userland/electron-builder/blob/master/packages/app-builder-lib/src/publish/KeygenPublisher.ts#L239). I'm sure they'd be more than willing to have an option of creating releases in a draft state instead. 

Would you be open to creating an issue and possibly a PR over at [election-userland/electron-builder](https://github.com/electron-userland/electron-builder)?

**Comment by JonHollander** [comment]
Sure. I'll create an issue and PR. Should be simple to add it.
